backend/controller/fundingitems.controller.js

```diff
@@ -29,11 +31,37 @@ const createFundingItemController = (req, res) => {
 }
 
 const updateFundingItemController = (req, res) => {
+    if (req.body.sf_link || req.body.ppr_links || req.body.upr_links) {
+        res.status(400).json(
+            'Error: sf_link, ppr_links, and upr_links in FI cannot be patched'
+        )
+        return
+    }
+
     updateFundingItem(req.params.id, req.body)
         .then((updatedFI) => res.status(200).json(updatedFI))
         .catch((err) => res.status(500).json('Error: ' + err))
 }
 
+const updateSFLinkFundingItemController = async (req, res) => {
+    const { sf_link } = req.body
```

Comment:
thoughts on just making this part of the endpoint? so it would be something like /:id/updatesxlink/:new_id?

Reply:
valid. let's do it